runtime/startup_aux.c

```diff
@@ -136,6 +131,19 @@ void caml_parse_ocamlrunparam(void)
   }
 }
 
+extern const char_os *caml_executable_ocamlrunparam;
+
+void caml_parse_ocamlrunparam(void)
```

Comment:
Why does this have to become global?

Reply:
It was already global, I'm introducing a function that takes the ocaml runtime parameters string as an argument in `static void parse_ocamlrunparam(const char_os *opt)`. See the change at line 91. Happy to turn it static, though!